browser/ui/views/frame/brave_contents_view_util.cc

```diff
@@ -76,7 +76,23 @@ gfx::RoundedCornersF BraveContentsViewUtil::GetRoundedCornersForContentsView(
     return rounded_corners;
   }
 
-  const bool is_sidebar_visible = browser_view->IsSidebarVisible();
+  bool show_vertical_tab = tabs::utils::ShouldShowVerticalTabs(
```

Comment:
Thinking out loud: I guess I should make `tabs::utils::**Vertical**` stuff a BrowserWindowFeature in the future, so that we can get rid of GetBrowserForMigrationOnly()..